hatchery/config.go

```diff
@@ -204,6 +201,25 @@ func LoadConfig(configFilePath string, loggerIn *log.Logger) (config *FullHatche
 		data.ContainersMap[hash] = container
 	}
 
+	if data.Config.LicenseUserMapsTable == "" {
+		data.Logger.Printf("Warning: no 'license-user-maps-dynamodb-table' in configuration: will be unable to store license-user-map data in DynamoDB")
+	} else if data.Config.LicenseUserMapsGSI == "" {
+		data.Logger.Printf("Error: dynamodb table present but missing 'license-user-maps-global-secondary-index'")
+	}
+
+	for _, container := range data.Config.Containers {
+		data.Logger.Printf("Checking license config info for container %s", container.Name)
+		if container.License.Enabled {
+			if data.Config.LicenseUserMapsTable == "" {
+				data.Logger.Printf("Error: no 'license-user-maps-dynamodb-table' in configuration but container is configured for license %s", container.Name)
+			}
+			ok := validateContainerLicenseInfo(container.Name, container.License)
+			if !ok {
+				data.Logger.Printf("Error: container '%s' has an invalid 'license' configuration.", container.Name)
```

Comment:
in a few places in this file, when there is a configuration error you don't exit, so the service will keep running. is that on purpose? should we crash when the validation fails?

Reply:
Yes, good idea!